phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/MainWindowView.java

```diff
@@ -74,16 +74,22 @@ protected void setSideBar(SideBar sideBar) {
         this.leftContent.setContent(sideBar);
     }
 
-    public void showRightView(Node nodeToShow) {
+    public void setCenter(Node nodeToShow) {
         this.content.setCenter(nodeToShow);
     }
 
+    public void showDetailsView(Node nodeToShow) { this.content.setRight(nodeToShow); }
```

Comment:
It may be nice to enlarge the miniature list widget again, after you've clicked on a category.
On the current master this is only possible through a category change, because only this clears the details view, doesn't it?